src/native/src/cmake/FindGLFW3.cmake

```diff
@@ -0,0 +1,46 @@
+# Locate the glfw3 library
+#
+# This module defines the following variables:
+#
+# GLFW3_LIBRARY      the name of the library;
+# GLFW3_INCLUDE_DIR  where to find glfw include files.
+# GLFW3_FOUND        true if both the GLFW3_LIBRARY and GLFW3_INCLUDE_DIR have been found.
+#
+# To help locate the library and include file, you can define a 
+# variable called GLFW3_ROOT which points to the root of the glfw library 
+# installation.
+#
+
+# default search dirs
+set( _glfw3_HEADER_SEARCH_DIRS 
+  "/usr/include"
+  "/usr/local/include"
+  "C:/Program Files (x86)/glfw/include" )
+set( _glfw3_LIB_SEARCH_DIRS
+  "/usr/lib"
+  "/usr/local/lib"
+  "C:/Program Files (x86)/glfw/lib-msvc110" )
```

Comment:
I propose http://pastebin.com/k9R9ayJ3 as an alternate FindGLFW3.cmake. It checks first if your have the GLFW3 source code and, if so, it adds GLFW3 as a subdirectory. If you don't have the GLFW3 source code, it does things your way. I still don't like that, on Windows, you only search for binaries for MSVC11, but this works a bit better.